narwhals/_arrow/expr.py

```diff
@@ -249,12 +249,18 @@ def arg_true(self) -> Self:
     def sample(
         self: Self,
         n: int | None = None,
-        fraction: float | None = None,
         *,
+        fraction: float | None = None,
         with_replacement: bool = False,
+        seed: int | None = None,
```

Comment:
Think np.random.Generator can be included here

Reply:
Cool thanks! I somehow keep forgetting about that! Just changed it